Polygon_mesh_processing/examples/Polygon_mesh_processing/CMakeLists.txt

```
@@ -119,5 +127,7 @@ create_single_source_cgal_program( "triangulate_faces_example_OM.cpp")
 target_link_libraries( triangulate_faces_example_OM PRIVATE ${OPENMESH_LIBRARIES} )
 endif(OpenMesh_FOUND)
 
-
+if(TARGET ceres)
+  target_link_libraries( mesh_smoothing_example PRIVATE ceres)
+endif(TARGET ceres)
```

I would even:
  - put down the `find_package(Ceres QUIET)`, here, just before its use,
  - remove the `if(CERES_FOUND)` block, and then, use `target_compile_definitions`, as below:

```suggestion
  target_compile_definitions( mesh_smoothing_example PRIVATE CGAL_PMP_USE_CERES_SOLVER)
endif(TARGET ceres)
```